docs/dapp/opl/README.md

```diff
@@ -5,19 +5,147 @@ empowers developers to add new functionality to smart contracts on the most
 popular EVM networks like Ethereum, BNB Chain, and Polygon with encrypted
 transactions and confidential state.
 
-## DApp Development
-
-The Oasis Privacy Layer contains a set of [smart contracts](https://github.com/oasisprotocol/sapphire-paratime/tree/main/contracts/contracts/opl)
-that integrate [Sapphire](../sapphire/README.mdx) into your existing and future
-Web3 applications, introducing confidentiality.
-
-Application developers on any EVM-compatible network can add confidential state
-and selective disclosures to their application through the usage of a smart
-contract deployed on OPL without migrating their application stack. OPL enabled
-smart contracts provide a default bridge for developers to pass messages
-between Sapphire and other networks.
+By using [a Solidity library](https://github.com/oasisprotocol/sapphire-paratime/tree/main/contracts/contracts/opl)
```

Comment:
Should we use the link to @oasisprotocol/sapphire-contracts npmjs.com package instead here?